cmd/environments/environments.go

```diff
@@ -17,12 +19,20 @@ func NewEnvironmentsCmd(
 		Long:  "Make requests (list, create, etc.) on environments",
 	}
 
-	getCmd, err := NewGetCmd(analyticsTracker, client)
+	getCmd, err := NewGetCmd(client)
 	if err != nil {
 		return nil, err
 	}
-
 	cmd.AddCommand(getCmd)
 
+	cmd.PersistentPreRun = func(cmd *cobra.Command, args []string) {
```

Comment:
Can you set this when we create `cmd` above? Otherwise it seems like it's doing something special that wouldn't work if it were set at the same time as the other fields.